submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+*We could also use assembling machine 3's for these processes, but they would consume more energy despite the speed gain.
+
+The total processing energy ("E") needed for a conversion can be calculated using the machine power usage ("P") multiplied by the total crafting time for all of the input ("t"). This total crafting time is the time taken for one crafting cycle ("t_c") multiplied by the number of crafting cycles ("n") to cover the entire input quantity, divided by the machine crafting speed ("C"). Hence we summarize processing energy consumption as:
+
+`E = P * t = P * t_c * n / C`
+
+Once we have the crafting recipes, fuel energy values, and the processing energy equation, we have all we need to make the conversion diagrams. Now let's look at them:
+
+## Coal vs. liquified coal
+
+The simple way to process coal is to just liquify it and immediately convert the outputs into solid fuel, as below:
+
+{% include image.html 
+
+src= 'src'
+ alt='Processing coal into solid fuel directly'
+ caption='Processing coal into solid fuel directly' 
+
+%}
+
+We find that 10 coal yields 5.75 solid fuel this way. Meanwhile, the coal processing requires energy for heating steam and running machines. We can calculate the total energy gain by subtracting all the input energy and processing energy from the output energy. Then, the percent gain is the energy gain divided by the energy obtained from directly burning the input fuel.
+
+`Energy gain = 69 - 40 - 1.5 - 2.1 - 1.365 - 0.84 - 0.21 = 22.985MJ`
+
+`Percent gain = 22.985MJ / 40MJ = 57.4625% `
+
+So we see that simply liquifying the coal gets us about 57% more energy out of it. Now, can we get out even more if we use cracking?
+
+## Adding cracking to coal liquification
+
+Cracking adds and extra step to the process, but looking at the recipes shows that it can increase solid fuel yield: Cracking heavy oil provides 30/40 or 75% as much light oil. Meanwhile, 10 light oil is enough to make 1 solid fuel rather than 20 heavy oil, meaning that it has 200% of the yield. Combining these, by cracking heavy oil we get 75% * 200% = 150% as much solid fuel as before. 
+
+However, the same cannot be said about cracking light oil. It provides 20/30 or about 66.7% as much petroleum gas, but also 20 petroleum gas is required for making solid fuel instead of 10 light oil, meaning 50% as much yield. Combining these, by cracking light oil we get 66.7% * 50% = 33.4% as much solid fuel as before, meaning that light oil cracking should be avoided.
```

Comment:
```suggestion
However, the same cannot be said about cracking light oil. It provides 20/30 or about 66.7% as much petroleum gas, but 20 petroleum gas is also required for making solid fuel instead of 10 light oil, meaning 50% as much yield. Combining these, by cracking light oil we get 66.7% * 50% = 33.4% as much solid fuel as before, meaning that light oil cracking should be avoided.
```

Reply:
"but also the recipe for solid fuel requires 20 petroleum gas rather than 10 light oil"